### DISCUSSION TITLE
Continuous playback of playlist

### DISCUSSION THREAD

**Discussion by gkertai** [videovillage/community#110] [Ideas]
When I am playing through a playlist, it would be much appreciated if Screen could automatically jump from the end of a clip to the beginning of the next (or the same in reverse) while preserving the current playback speed. Currently you need to keep clicking ↓ at the end of each file. This really becomes a hassle when you'd want to fast forward through many files to find something, then it's ↓ followed by LLL to get it up to speed again.

I suppose a Playback Behavior called *Playlist* (or similar) could handle this nicely. 

![Screenshot 2021-04-01 at 15 11 22](https://user-images.githubusercontent.com/1874498/113298909-9be7b680-92fc-11eb-9b3c-b987f7fd0102.png)

**Comment by gregcotten** [comment]
I agree this is needed!

**Comment by travisbutton** [comment]
Yeah a "Playlist" is not really a playlist if it doesn't play the contents as a stringout. It should match the playback functionality of "Sequence" in RV.
